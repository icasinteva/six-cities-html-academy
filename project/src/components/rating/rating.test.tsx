import { render } from '@testing-library/react';

import Rating from './rating';

describe('Component: Rating', () => {
  it('should render correctly', () => {
    const { container } = render(<Rating className='property' rating={4.8} />);

    expect(container).toMatchSnapshot();
  });
});

