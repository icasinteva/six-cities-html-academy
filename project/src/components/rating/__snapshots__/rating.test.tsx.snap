// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Rating should render correctly 1`] = `
<div>
  <div
    class="property__rating rating"
  >
    <div
      class="property__stars rating__stars"
    >
      <span
        data-testid="rating"
        style="width: 100%;"
      />
      <span
        class="visually-hidden"
      >
        Rating
      </span>
    </div>
  </div>
</div>
`;
